Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap

```diff
@@ -1,19 +1,33 @@
 itk_wrap_class("itk::PyVectorContainer")
+  UNIQUE(identifiertypes "${WRAP_ITK_SCALAR};UC;UI;UL;ULL")
   UNIQUE(types "${WRAP_ITK_SCALAR};D;UI")
+
+  unique(real_types "${WRAP_ITK_REAL};D")
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
-    # Wrap itk.Point[itk.F, ${d}]
-    list(APPEND types PF${d})
+    # Wrap itk.Offset[${d}]
+    list(APPEND types O${d})
+    # Wrap itk.ContinuousIndex[itk.D,${d}]
+    list(APPEND types CID${d})
+    foreach(t ${real_types})
+      # Wrap itk.Point[${t}, ${d}]
+      list(APPEND types P${t}${d})
+      # Wrap itk.Vector[${t}, ${d}]
```

Comment:
If we want `itk.Vector` do we want `itk.CovariantVector` too?

Reply:
`VectorContainer` is not currently wrapped for `CovariantVector`, should we add that wrapping?

Reply:
I don't see wrapping for Vector or CovariantVector in `itkVectorContainer.wrap` so that is self consistent.  The only place I see wrapping for Vector in `itkPyVectorContainer.wrap` is the new lines 14 & 15 in this pull request:
```cmake
# Wrap itk.Vector[${t}, ${d}]
list(APPEND types V${t}${d})
```
I am not sure I understand the motivation there, so I am not sure whether CovariantVector is also needed.  If it is, the code would be something like:
```cmake
# Wrap itk.Vector[${t}, ${d}] and itk.CovariantVector[${t}, ${d}]
list(APPEND types V${t}${d})
list(APPEND types CV${t}${d})
```

Reply:
Let's avoid wrapping `CovariantVector` until it is needed.

Reply:
Following up briefly, `itk::VectorContainer` is wrapped for `itk::Vector` in [`itkVectorContainer.wrap`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/wrapping/itkVectorContainer.wrap#L16):
>  itk_wrap_template("${ITKM_IT}${ITKM_V${t}${d}}" "${ITKT_IT},${ITKT_V${t}${d}}")

The primary motivation for updating `PyVectorContainer` wrapping here is to improve ease of access to Python array representation of `VectorContainer` objects. `itk.array_from_vector_container` must translate an input `VectorContainer` to an equivalently wrapped `PyVectorContainer` prior to representation as a numpy Array. The goal is to wrap `PyVectorContainer` most nearly for common use cases where we might try to translate a `VectorContainer` to numpy.